tests/test_calculator.hh

```diff
@@ -569,9 +517,12 @@ namespace rascal {
         {{"type", "Constant"},
          {"gaussian_sigma", {{"value", 0.4}, {"unit", "AA"}}}}};
     std::vector<json> radial_contribution_hypers{{{"type", "GTO"}}};
+    // std::vector<json> rep_hypers{
+    //     {{"max_radial", 2}, {"max_angular", 2}, {"compute_gradients", true}},
+    //     {{"max_radial", 3}, {"max_angular", 0}, {"compute_gradients",
+    //     true}}};
     std::vector<json> rep_hypers{
-        {{"max_radial", 2}, {"max_angular", 2}, {"compute_gradients", true}},
-        {{"max_radial", 3}, {"max_angular", 0}, {"compute_gradients", true}}};
```

Comment:
Is this the reason the test got faster? Less computations?

Reply:
I am not sure actually